universal-application-tool-0.0.1/app/views/admin/programs/ProgramBlockEditView.java

```diff
@@ -334,6 +334,7 @@ private ContainerTag questionBankPanel(
         new QuestionBank()
             .setQuestionAction(addQuestionAction)
             .setCsrfTag(csrfTag)
+            .setQuestionBankJsTag(layout.viewUtils.makeLocalJsTag("questionBank"))
```

Comment:
Instead of passing it into `QuestionBank` just pass the bundle into this function and use `bundle.addFooterScripts`

Reply:
thanks! i was wondering how to do this good.

Reply:
is there a reason the `preview.ts` script isn't done that way? these are admin-only scripts.

Reply:
I moved `preview` and `questionBank` into the `adminLayout` bundle's footer scripts.

Reply:
`preview` was added only if we were on the add/edit question page, but that's probably an unnecessary optimization.